README.md

```diff
@@ -41,23 +37,24 @@
   </ul>
 </details>
 
-<!-- Features -->
+<!-- Links -->
 
-### Key Features <a name="key-features"></a>
+### Live Link <a name="live-link"></a>
 
-- *Model, View and Controller (MVC)*
-- *Devise, Authentication, Authorization and Access Management*
+[Live link](https://bofulo-budget-app.onrender.com/)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- Video Demo Link -->
 
-<!-- Live Link -->
+[Video link](https://www.loom.com/share/44db4ea9ac314739b420f605ae223ecb) 
```

Comment:
- [ ] [OPTIONAL] Excellent presentation!
It will be nice to present the mobile version of the application next time since this is a mobile design.